digitaltwin_client/samples/readme.md

```diff
@@ -4,24 +4,98 @@ This directory contains samples demonstrating creating and using Digital Twin in
+
+  From the cmake directory:
+  ```
+  cmake --build .
+  ```
+  This will build the `digitaltwin_sample_device` executable along with the aforementioned sample interfaces.
+
+* Copy the Connection String of the Azure IoT Hub device you wish to enable for Digital Twin:
+
+* Run the sample.
+
+  From the cmake directory:
+  
+  **If using Windows:** 
+
+  ```
+  cd .\digitaltwin_client\samples\digitaltwin_sample_device\Debug\
+  ```
+  Run the executable using your connection string, e.g.:
+  ```
+  .\digitaltwin_sample_device.exe "HostName=yourHub;DeviceId=yourDigitalTwinDeviceId;SharedAccessKey=secret"
+  ```
+
+  **If using Linux:**
+
+  ```
+  cd digitaltwin_client/samples/digitaltwin_sample_device
+  ```
+  Run the executable using your connection string, e.g.:
+  ```
+  ./digitaltwin_sample_device "HostName=yourHub;DeviceId=yourDigitalTwinDeviceId;SharedAccessKey=secret"
+  ```
+
+  Note: In both OS options, be sure to include quotations around the Connection String.
+
+
 
-* If you wish to use `digitaltwin_sample_ll_device`, see  [digitaltwin_sample_ll_device](./digitaltwin_sample_ll_device/readme.md) for additional instructions on running the \_LL\_ sample.
+### digitaltwin\_sample\_ll_device
 
-* Build [digitaltwin_sample_device](digitaltwin_sample_device) directory.  
-This will build the `digitaltwin_sample_device` executable along with the sample interfaces that it uses (which are also contained under the ./samples directory.  The libraries implementing the interfaces (`digitaltwin_sample_device_info` and `digitaltwin_sample_environmental_sensor`) implement helper functions that `digitaltwin_sample_device` invokes to perform basic DigitalTwin operations.
+* Configure the files.
 
-* Run `digitaltwin_sample_device` [connectionString], where connectionString is the connection string of an IoTHub device you wish to DigitalTwin enable.  For example, *digitaltwin_sample_device HostName=yourHub;DeviceId=yourDigitalTwinDeviceId;SharedAccessKey=secret*.
+  Please go [here](./digitaltwin_sample_ll_device) to follow the configuration steps before building the SDK.
+
+* Use cmake to build the [digitaltwin\_sample\_ll_device](digitaltwin_sample_ll_device) directory.
+
+  ```
+  cmake --build .
+  ```
+  This will build the `digitaltwin_sample_ll_device` executable along with the aforementioned sample interfaces.
+  
+* Run the sample.
+
+  From the cmake directory:
+
+  **If using Windows:** 
+  ```
+  cd .\digitaltwin_client\samples\digitaltwin_sample_ll_device\Debug\
+
+  .\digitaltwin_sample_ll_device.exe ..\..\..\..\..\digitaltwin_client\samples\digitaltwin_sample_ll_device\sample_config\dpsSymmKey.json
+  ```
+
+  **If using Linux:**
+  ```
+  cd digitaltwin_client/samples/digitaltwin_sample_ll_device/
+  
+  ./digitaltwin_sample_ll_device dpsSymmKey.json ../../../../../digitaltwin_client/samples/digitaltwin_sample_ll_device/sample_config/dpsSymmKey.json
```

Comment:
"./digitaltwin_sample_ll_device dpsSymmKey.json .." - remove the first dpsSymmKey.json that comes immediately after digitaltwin_sample_ll_device, since the real source of this is the ..